bpython/cli.py

```diff
@@ -160,6 +161,7 @@ def __init__(self, interface):
         self.encoding = getpreferredencoding()
         self.interface = interface
         self.buffer = list()
+        self.errors = "strict"
```

Comment:
It should still be a separate commit. Please remove it. It has nothing to do with the new feature.